gensim/sklearn_integration/SklearnWrapperGensimLdaModel.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+#
+"""
+scikit learn interface for gensim for easy use of gensim with scikit-learn
+follows on scikit learn API conventions
+"""
+from gensim import models
+from gensim import matutils
+from scipy.sparse.csr import csr_matrix
+
+
+class SklearnWrapperLdaModel(models.LdaModel,object):
```

Comment:
PEP8: space after comma.

Actually, not relevant at all, because `LdaModel` already inherits from object naturally.